lib/improver/nowcasting/optical_flow.py

```diff
@@ -242,6 +242,11 @@ def _make_subboxes(self, field):
                 **weights** (np.ndarray):
                     1D numpy array containing weights values associated with
                     each listed box.
+
+        References:
+            Bowler, N., Pierce, C. and Seed, A. 2004: Development of a
+            precipitation nowcasting algorithm based upon optical flow
+            techniques. Journal of Hydrology, 288, 74-91.
```

Comment:
This is referenced at the top of the plugin, do we need the repeat? You could consider moving it out of the __init__ doc string and into the generic plugin doc string, but I don't think we need repeats throughout.

Reply:
I've moved all the references to the top level plugin documentation